src/common/ITRoleRouter.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+
+interface ITProffesion {
+  proffesion: string;
+  path: string;
+}
+
+export const ITRoleRouter: React.FC<ITProffesion> = ({ proffesion, path }) => (
+  <Route exact path={path} key={proffesion}>
```

Comment:
Just wondering: are going to have a situation where `path` and `profession` props will be different? Maybe one prop would be enough - what do you think?